### ISSUE TITLE
Make sure test-failing screen shots make it into CircleCIs artifacts

### ISSUE THREAD

**Issue by mccalluc** [pulibrary/tiger-data-app#71]
For [example](https://app.circleci.com/pipelines/github/pulibrary/tiger-data-app/102/workflows/c62a06e5-aaa7-4e93-b15e-acabf48c8b08/jobs/357/steps?invite=true#step-110-39)
```
     [Screenshot Image]: /home/circleci/tiger_data/tmp/capybara/failures_r_spec_example_groups_application_accessibility_when_browsing_the_homepage_complies_with_wcag_2_0_aa_and_section_508_872.png
```
But that file isn't viewable: right now, the only artifacts being captured are the coverage report. I remember adding a configuration for this to pdc_describe, so it shouldn't be hard.
